Source/WebCore/platform/LocalizedStrings.cpp

```diff
@@ -503,6 +503,13 @@ String contextMenuItemTagTranslate(const String& selectedString)
 }
 #endif
 
+#if ENABLE(UNIFIED_PDF)
+String contextMenuItemPDFOpenWith(const String& app)
```

Comment:
I don't think there's any case when `app != "Preview"`, right? We could just

```cpp
#if ENABLE(UNIFIED_PDF)
String contextMenuItemPDFOpenWithPreview()
{
    return WEB_UI_STRING("Open with Preview", "Open with Preview context menu item");
}
#endif
```

Reply:
Alternatively if you keep the current definition, you should also format the description argument because it will actually be `Open with {app} context menu item`.

Reply:
Ah yeah good catch! I just ended up creating the string like you demonstrated